tls/test_tls_stress.py

```diff
@@ -184,17 +188,50 @@ def check_alg(self, alg):
 
         self.assertFalse(tls_perf.stderr)
 
-    def test_DHE_RSA_AES128_GCM_SHA256(self):
-        self.check_alg("tls-perf-DHE-RSA-AES128-GCM-SHA256")
-
-    def test_DHE_RSA_AES256_GCM_SHA384(self):
-        self.check_alg("tls-perf-DHE-RSA-AES256-GCM-SHA384")
-
-    def test_DHE_RSA_AES128_CCM(self):
-        self.check_alg("tls-perf-DHE-RSA-AES128-CCM")
+    @marks.Parameterize.expand(
+        [
+            marks.Param(name="DHE-RSA-AES128-GCM-SHA256", alg="tls-perf-DHE-RSA-AES128-GCM-SHA256"),
+            marks.Param(name="DHE-RSA-AES256-GCM-SHA384", alg="tls-perf-DHE-RSA-AES256-GCM-SHA384"),
+            marks.Param(name="DHE-RSA-AES128-CCM", alg="tls-perf-DHE-RSA-AES128-CCM"),
+            marks.Param(name="DHE-RSA-AES256-CCM", alg="tls-perf-DHE-RSA-AES256-CCM"),
+        ]
+    )
+    def test(self, name, alg):
+        self.check_alg(alg)
+
+    def __reload_tempesta(self):
+        tempesta: Tempesta = self.get_tempesta()
+        while not self.stop_flag:
+            tempesta.run_start()
```

Comment:
Why do you use `run_start` here? I don't understand like it works. Why don't you use `restart` method or `reload`?